src/frontend/screens/MapScreen/index.tsx

```diff
@@ -172,12 +172,8 @@ export const MapScreen = ({
           {isFinishedLoadingStyle && authState !== 'obscured' && (
             <>
               <RemoteDetectionAlertsMapLayer />
-              {isTracking && (
-                <>
-                  <CurrentTrackMapLayer />
```

Comment:
this was causing a race condition and causing [this error](https://github.com/digidem/comapeo-mobile/pull/1390#discussion_r2356680772):

Essentially endTracking was being called, which sets isTracking to be false. The location tries to update the layer, but react tears it down, causing mapbox to say " message=Source routeSource is not in style" aka does not exist. 

Since this map layer is based on persisted state of the location in tracks, which are cleared when there are no tracks, we dont need to conditionally render it with this check here